c/extras/py_ftrl.cc

```diff
@@ -610,7 +578,7 @@ Returns
 )");
 
 
-void Ftrl::reset(const NoArgs&) {
+void Ftrl::reset(const PKArgs&) {
```

Comment:
So there is no `NoArgs` anymore?

Reply:
Yeah, I don't think it has any use really: you can always declare `PKArgs(0,0,0,false,false)`. On the other hand, removing `NoArgs` and merging `PKArgs` with parent `Args` will simplify the class hierarchy and avoid virtual methods.

Reply:
I've used `PKArgs(0,0,0,false,false)` some time ago, but then we changed that part to `NoArgs`. At some point I realized it was a good indicator that method takes no parameters, one don't need to go into details and can get such an info just from the `*Args` name. Not a big deal though...